init.lua

```diff
@@ -7,12 +7,12 @@ local function add_tool(name, func)
 end
 
 minetest.register_on_dignode(function(_, oldnode, digger)
-	if digger == nil then
+	if not digger
```

Comment:
not digger ist es wen digger nichts (nil) ist oder falsch (false)
digger ist nie falsch, aber er/sie/es kann nichts sein